templates/partials/footer.tmpl

```diff
@@ -14,31 +15,34 @@
                                 </li>
 
                                 <li class="footer-nav__item">
-                                    <a href="https://www.ons.gov.uk/cookies">Cookies</a>
+                                    <a href="/cookies">Cookies</a>
```

Comment:
The footer links need to stay as absolute paths, the developer site sits on a different subdomain